test/kubernetes_unittest.cc

```diff
@@ -837,45 +905,26 @@ TEST_F(KubernetesTestWithInstance, GetPodAndContainerMetadata) {
     {"pod_name", "TestPodName"},
   }), m[1].resource());
   EXPECT_FALSE(m[1].metadata().ignore);
-  EXPECT_EQ("1.6", m[1].metadata().version);
+  EXPECT_EQ("1.6", m[1].metadata().version);  // Hard-coded in kubernetes.cc.
   EXPECT_FALSE(m[1].metadata().is_deleted);
   EXPECT_EQ(time::rfc3339::FromString("2018-03-03T01:23:45.678901234Z"),
             m[1].metadata().created_at);
   EXPECT_EQ(Timestamp(), m[1].metadata().collected_at);
-  json::value container_metadata = json::object({
-    {"blobs", json::object({
-      {"association", json::object({
-        {"raw", json::object({
-          {"controllers", json::object({
-            {"topLevelControllerName", json::string("TestPodName")},
-            {"topLevelControllerType", json::string("Pod")},
-          })},
-          {"infrastructureResource", json::object({
-            {"labels", json::object({
-              {"instance_id", json::string("TestID")},
-              {"zone", json::string("TestZone")},
-            })},
-            {"type", json::string("gce_instance")},
-          })},
-          {"nodeName", json::string("TestSpecNodeName")},
-        })},
-        {"version", json::string("TestVersion")},
-      })},
-      {"spec", json::object({
-        {"raw", json::object({
-          {"name", json::string("TestContainerName0")},
-        })},
-        {"version", json::string("1.6")},
-      })},
-      {"status", json::object({
-        {"raw", json::object({
-          {"name", json::string("TestContainerName0")},
-        })},
-        {"version", json::string("1.6")},
-      })},
+  json::value spec = json::object({
```

Comment:
You can inline these values into the `ContainerMetadataWithAssociations` call below...

Reply:
Done.